packages/core/src/Stores/notification-store.js

```diff
@@ -345,6 +345,10 @@ export default class NotificationStore extends BaseStore {
                     (needs_verification.includes('document') || document?.status !== 'verified');
                 const needs_poi = is_10k_withdrawal_limit_reached && identity?.status !== 'verified';
                 const onfido_submissions_left = identity?.services.onfido.submissions_left;
+                const poo_required =
+                    needs_verification?.includes('ownership') && ownership?.status?.toLowerCase() !== 'rejected';
```

Comment:
is there any `required` status that we can use it instead of this !==rejected status?

Reply:
@ali-hosseini-fs, No. There is no `required` status. Below are all the statuses we get from `BE`

![POO statuses](https://user-images.githubusercontent.com/104334373/201819192-abb6c832-3f53-4a56-a947-68d29f24da10.png)